tests/configuration/pages/pmmInventoryPage.js

```diff
@@ -334,4 +335,19 @@ module.exports = {
     I.click(this.fields.agentsLink);
     I.waitForVisible(agent, 30);
   },
+
+  async checkAgentsPresent(expectedAgentIds) {
+    const actualAgentIds = (await I.grabTextFromAll(this.fields.agentId))
+      .map((string) => string.replace('/agent_id/', ''));
+
+    if (expectedAgentIds.length !== actualAgentIds.length) {
+      assert.fail(`The number of actual Agents doesn't match expected (Expected ${expectedAgentIds.length} but got ${actualAgentIds.length})`);
+    }
+
+    expectedAgentIds.forEach((agentId) => {
+      if (!actualAgentIds.includes(agentId)) {
```

Comment:
This could be just `assertTrue()` or if the aim is not to use "asserts" here than it should be `throw new Exception();`

Reply:
fixed

Reply:
awesome! thanks!